src/diffusers/models/unets/unet_1d.py

```diff
@@ -101,7 +101,7 @@ def __init__(
         # time
         if time_embedding_type == "fourier":
             self.time_proj = GaussianFourierProjection(
-                embedding_size=8, set_W_to_weight=False, log=False, flip_sin_to_cos=flip_sin_to_cos
+                embedding_size=block_out_channels[0], set_W_to_weight=False, log=False, flip_sin_to_cos=flip_sin_to_cos
```

Comment:
I think it might be better to introduce a `time_embedding_dim` argument to the class init and configure this way
https://github.com/huggingface/diffusers/blob/f8ba5cd77ace516a7bfc22551e6a9a93befe4a3a/src/diffusers/models/unets/unet_2d_condition.py#L549